schema/dex/trades.sql

```diff
@@ -157,6 +158,51 @@ WITH rows AS (
 
         UNION
 
+        --- Kyber_V2
+        SELECT
+            evt_block_time AS block_time,
+            'Kyber' AS project,
+            NULL AS VERSION,
+            src AS token_a_address,
+            dest AS token_b_address,
+            kyber_v2."Network_evt_KyberTrade".contract_address exchange_contract_address,
+            evt_tx_hash AS tx_hash,
+            ethereum.transactions.from AS trader_a,
+            NULL::bytea AS trader_b,
+            NULL::integer[] AS trace_address,
+            evt_index AS evt_index,
+
+            (SELECT SUM(a)
+                    FROM UNNEST(CASE
+                                    WHEN src_token.contract_address = '\xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' THEN "e2tSrcAmounts"
+                                    ELSE "t2eSrcAmounts"
+                                END) AS a) AS token_a_amount_raw,
+
+            (SELECT SUM(s)
+            FROM UNNEST(ARRAY
```

Comment:
Volumes produced are way too high. See this query: https://explore.duneanalytics.com/queries/4376/source#8523 with what our DEX dashboard volume query will produce as yesterdays volume. Far higher than https://tracker.kyber.network/#/

I suspect this part might cause it but can't really wrap my head around it.

Reply:
Kay, let me check it again.

Reply:
This part should be divided into token - eth and eth - token. I have updated the query. It looks more reasonable now: https://explore.duneanalytics.com/queries/6612/source#13133